app/models/user.rb

```diff
@@ -1,8 +1,12 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable and :omniauthable
+  devise :database_authenticatable, 
+         :recoverable, :rememberable, :trackable, :validatable
   include SnoopDogg
```

Comment:
@PaulStorberg Put included concerns above everything else, directly below the class name and without space between the class names and the concern

Reply:
complete